src/components/OSCALLoader.js

```diff
@@ -100,6 +103,13 @@ export default function OSCALLoader(props) {
   );
 }
 
+function getRequestedUrl(){
+  /*Returns url parameter provided by the browser url, if
+    it exists. If the url parameter exists, we want to
+    override the default viewer url.*/
```

Comment:
This doesn't seem to match our conventions for comments. Consider:

```suggestion
  // Returns url parameter provided by the browser url, if
  //  it exists. If the url parameter exists, we want to
  //  override the default viewer url.
```

or 
```suggestion
  /* 
   * Returns url parameter provided by the browser url, if
   * it exists. If the url parameter exists, we want to
   * override the default viewer url.
   */
```

I will say that looking at the results of:
```
$ git grep -A2 '/\*[^*]' src/ # Find comments that start with `/*` but not `/**` and the two lines after
$ git grep -A2 '//' src/ # Find comments that start with `//` and the two lines after
```

The `//` style is far more common in the code base which makes sense because it makes it easier to comment out large blocks of code for testing (because `//` comments can be nested in `/*` comments but `/*` comments cannot be nested in `//` comments).

Reply:
@mikeisen1 it was mentioned above but I think it is generally better to address the comment before resolving conversation. Just so it doesn't get lost in this large thread.